src/Utils/utils.ts

```diff
@@ -33,9 +34,19 @@ export const formatTimeShort = (time: Time) => {
 
 export const relativeDate = (date: DateLike, withoutSuffix = false) => {
   const obj = dayjs(date);
-  return `${obj.fromNow(withoutSuffix)}${
-    withoutSuffix ? " ago " : ""
-  } at ${obj.format(TIME_FORMAT)}`;
+  const isToday = obj.isSame(dayjs(), "day");
+
+  const relative = obj.fromNow(withoutSuffix);
+
+  const hasTime = obj.hour() !== 0 || obj.minute() !== 0 || obj.second() !== 0;
```

Comment:
consider simplifying the logic

```suggestion
  const hasTime = !!(obj.hour() || obj.minute() || obj.second());
```